<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>movementsCheckingWidget</class>
 <widget class="QWidget" name="movementsCheckingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pointer les mouvements</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0" colspan="5">
      <widget class="QTableView" name="tableView"/>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="endDateLabel">
       <property name="text">
        <string>Date fin</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="userLabel">
       <property name="text">
        <string>Utilisateur</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="recordButton">
       <property name="text">
        <string>Enregi&amp;stre</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" rowspan="3" colspan="2">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>270</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QDateEdit" name="dateEndEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="beginDateLabel">
       <property name="text">
        <string>Date d√©but</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>&amp;Quitter</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="dateBeginEdit"/>
     </item>
     <item row="5" column="2" colspan="2">
      <widget class="QComboBox" name="comboBoxUsers"/>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="labelAlpha">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
